src/api-reference/expense/expense-report/v4.allocations.markdown

```diff
@@ -25,22 +27,24 @@ Access to this documentation does not provide access to the API.
 
 Required Scopes:
 
-|Name|Description|Endpoint|
-|---|---|---|
-|`expense.report.read`|Get information about expense reports.|GET|
-|`expense.report.readwrite`|Read and write expense report headers.        |PATCH|
-|`user.read`|Get User Information, necessary for `userID`.|GET|
+| Name                       | Description                                   | Endpoint |
+|----------------------------|-----------------------------------------------|----------|
```

Comment:
Every single table was reformatted to line up perfectly, makes viewing/editing the actual markdown file much easier.